Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'puma', '~> 3.7'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rubocop', require: false
```

Comment:
We usually lock the gem version in order to keep them under control, the prefered way to do that is with the Pessimistic Operator, an example:

```
gem 'rubocop', '~> 0.59'
```